doc/releases/changelog-0.37.0.md

```diff
@@ -4,59 +4,172 @@
 
 <h3>New features since last release</h3>
 
-<h4>Execute faster with Default Tensor 🔗</h4>
+<h4>Execute wide circuits with Default Tensor 🔗</h4>
 
-* The `default.tensor` device now supports the `tn` method to simulate quantum circuits using exact tensor networks.
+* A new `default.tensor` device is now available for performing
+  [tensor network](https://en.wikipedia.org/wiki/Tensor_network) and
+  [matrix product state](https://en.wikipedia.org/wiki/Matrix_product_state) simulations
+  of quantum circuits using the
+  [quimb backend](https://quimb.readthedocs.io/en/latest/).
+  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  [(#5744)](https://github.com/PennyLaneAI/pennylane/pull/5744)
   [(#5786)](https://github.com/PennyLaneAI/pennylane/pull/5786)
+  [(#5795)](https://github.com/PennyLaneAI/pennylane/pull/5795)
 
-* The `default.tensor` device is introduced to perform tensor network simulations of quantum circuits using the `mps` (Matrix Product State) method.
-  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  Either method can be selected when instantiating the `default.tensor` device by setting the
+  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
+  device can simulate a large number of qubits with sensible approximations. Take this example that
+  can simulate 1000 qubits in a few seconds!
```

Comment:
```suggestion
  Either method can be selected when instantiating the `default.tensor` device by setting the
  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
  device can simulate a large number of qubits. Take this example that
  simulates 1000 qubits with a shallow circuit in a few seconds!
```

perhaps like this?
Claiming that it can _efficiently_ simulate a large number of qubits without saying any caveats could be misinterpreted I think

Reply:
:shipit: